dnf-behave-tests/fixtures/specs/kernel/kernel-3.0.0-1.fc29.spec

```diff
@@ -0,0 +1,59 @@
+Name:           kernel
+Epoch:          0
+Version:        3.0.0
+Release:        1.fc29
+
+License:        GPLv2 and Redistributable, no modification permitted
+URL:            https://www.kernel.org/
+
+Summary:        The Linux kernel
+
+Provides:       kernel = %{version}-%{release}
+Provides:       kernel(x86-64) = %{version}-%{release}
+
+Requires:       kernel-modules-uname-r = %{version}-%{release}.x86_64
+Requires:       kernel-core-uname-r = %{version}-%{release}.x86_64
+
+%description
+The kernel meta package
+
+%package core
+Summary:        The Linux kernel
+
+Provides:       installonlypkg(kernel)
+Provides:       kernel = %{version}-%{release}
+Provides:       kernel-uname-r = %{version}-%{release}.x86_64
+Provides:       kernel-core-uname-r = %{version}-%{release}.x86_64
+Provides:       kernel-drm-nouveau = 16
+Provides:       kernel-core = %{version}-%{release}
+Provides:       kernel-core(x86-64) = %{version}-%{release}
+Provides:       kernel-x86_64 = %{version}-%{release}
+
+%description core
+The kernel package contains the Linux kernel (vmlinuz), the core of any
+Linux operating system.  The kernel handles the basic functions
+of the operating system: memory allocation, process allocation, device
+input and output, etc.
+
+%package modules
+Summary:        kernel modules to match the core kernel
+
+Provides:       installonlypkg(kernel-module)
+Provides:       kernel-modules-uname-r = %{version}-%{release}.x86_64
+Provides:       kernel-modules = %{version}-%{release}
```

Comment:
I believe that `Provides:       kernel-modules = %{version}-%{release}` is redundant.